src/main/java/com/salesforce/dataloader/mapping/LoadMapper.java

```diff
@@ -88,8 +92,8 @@ public void verifyMappingsAreValid() throws MappingInitializationException {
         for (Map.Entry<String, String> entry : getMappingWithUnmappedColumns(false).entrySet()) {
             String sfdcName = entry.getValue();
             if(StringUtils.hasText(sfdcName)) {
-                final Field f = getClient().getField(sfdcName);
-                if (f == null)
+                final HashSet<Field> fieldSet = getClient().getField(sfdcName);
+                if (fieldSet == null | fieldSet.isEmpty())
```

Comment:
So with a fresh pull from forcedotcom/dataloader, and building it with

mvn clean package -license.skip=true

I get "Tests run: 285, Failures: 210, Errors: 9, Skipped: 0"

I must be missing something in my configuration...